producer/src/main/kotlin/streams/StreamsExtensionFactory.kt

```diff
@@ -57,7 +61,9 @@ class StreamsEventRouterLifecycle(private val db: GraphDatabaseAPI,
             StreamsProcedures.registerEventRouter(eventRouter = streamHandler)
             StreamsProcedures.registerEventRouterConfiguration(eventRouterConfiguration = streamsEventRouterConfiguration)
             streamHandler.start()
-            streamHandler.printInvalidTopics()
+            if (configuration.isSourceGloballyEnabled() && KafkaValidationUtils.isAutoCreateTopicsEnabled(KafkaConfiguration.from(configuration.config).asProperties())) {
+                streamHandler.printInvalidTopics()
```

Comment:
scusa non ha più senso metterlo dopo linea 75 dello stesso file?

Reply:
sì hai ragione...e oltretutto manca un "!" davanti alla seconda condizione. Perchè il log dev'essere stampato nel caso in cui l'auto-creazione sia disabilitata.